### ISSUE TITLE
Headings in Admin Interface Improperly Formatted

### ISSUE THREAD

**Issue by avaaec** [pressbooks/pressbooks#1774]
### Steps to Reproduce

1. Navigate to admin interface for a book
2. Navigate to the "My Catalog" page
3. Inspect the headings for the page and notice that the headings span the entire line instead of the just the phrase that is visually styled to look like a heading
4. For example the Heading 1 is "My Catalog Edit Profile Visit Catalog"

**Expected behavior:** 
Headings should be concise and make semantic sense for the content they head. The text that is visually styled to look like a heading should match the heading element in the DOM.
**Actual behavior:** 
The heading tags are not closed until the end of the line, causing all other text elements in the line to be appended to the heading text. Headings are semantically very different from there visual representation.

**Comment by laurenray** [comment]
This is UW priority 1 issue

**Comment by SteelWagstaff** [comment]
See https://github.com/pressbooks/pressbooks/blob/2965e63f1d50261ca9fc081d2ef1730255e8b764/inc/admin/class-catalog-list-table.php#L461-L464

**Comment by connermwallace** [comment]
Hi, I am Conner, a student working at the University of Washington Accessibility Technology Services. We have inspected this issue and verified that it has been fixed. Version: 5.12.0
